src/Microsoft.HttpRepl/Commands/HelpCommand.cs

```diff
@@ -221,49 +222,56 @@ public void CoreGetHelp(IShellState shellState, ICommandDispatcher<HttpState, IC
+            output.AppendLine($"{"ls",navCommandColumn}{dispatcher.GetCommand<ListCommand>().GetHelpSummary(shellState, programState)}");
+            output.AppendLine($"{"cd",navCommandColumn}{dispatcher.GetCommand<ChangeDirectoryCommand>().GetHelpSummary(shellState, programState)}");
 
             output.AppendLine();
             output.AppendLine("Shell Commands:".Bold().Cyan());
             output.AppendLine("Use these commands to interact with the REPL shell.");
             output.AppendLine();
 
-            output.AppendLine($"{"clear",navCommandColumn}{"Removes all text from the shell."}");
-            output.AppendLine($"{"echo [on/off]",navCommandColumn}{"Turns request echoing on or off, show the request that was made when using request commands."}");
-            output.AppendLine($"{"exit",navCommandColumn}{"Exit the shell."}");
+            output.AppendLine($"{"clear",navCommandColumn}{dispatcher.GetCommand<ClearCommand>().GetHelpSummary(shellState, programState)}");
+            output.AppendLine($"{"echo [on/off]",navCommandColumn}{dispatcher.GetCommand<EchoCommand>().GetHelpSummary(shellState, programState)}");
+            output.AppendLine($"{"exit",navCommandColumn}{dispatcher.GetCommand<ExitCommand>().GetHelpSummary(shellState, programState)}");
 
             output.AppendLine();
             output.AppendLine("REPL Customization Commands:".Bold().Cyan());
             output.AppendLine("Use these commands to customize the REPL behavior.");
             output.AppendLine();
 
-            output.AppendLine($"{"pref [get/set]",navCommandColumn}{"Allows viewing or changing preferences, e.g. 'pref set editor.command.default 'C:\\Program Files\\Microsoft VS Code\\Code.exe'`"}");
-            output.AppendLine($"{"run",navCommandColumn}{"Runs the script at the given path. A script is a set of commands that can be typed with one command per line."}");
-            output.AppendLine($"{"ui",navCommandColumn}{"Displays the Swagger UI page, if available, in the default browser."}");
+            output.AppendLine($"{"pref [get/set]",navCommandColumn}{dispatcher.GetCommand<PrefCommand>().GetHelpSummary(shellState, programState)}");
+            output.AppendLine($"{"run",navCommandColumn}{dispatcher.GetCommand<RunCommand>().GetHelpSummary(shellState, programState)}");
+            output.AppendLine($"{"ui",navCommandColumn}{dispatcher.GetCommand<UICommand>().GetHelpSummary(shellState, programState)}");
             output.AppendLine();
             output.AppendLine("Use help <COMMAND> to learn more details about individual commands. e.g. `help get`".Bold().Cyan());
             output.AppendLine();
 
             shellState.ConsoleManager.Write(output.ToString());
         }
+
+        private static T GetCommand<T, TState, TParseResult>(ICommandDispatcher<TState, TParseResult> dispatcher) where T: class, ICommand where TParseResult: ICoreParseResult
```

Comment:
Is this used?  It looks like all the calls are using the extension method.

Reply:
Whoops. No, that was a precursor to the extension method that I forgot to remove. Removed in df3f30f.